model/src/trade.rs

```diff
@@ -5,7 +5,7 @@ use num_bigint::BigUint;
 use primitive_types::{H160, H256};
 use serde::{Deserialize, Serialize};
 
-#[derive(Eq, PartialEq, Clone, Debug, Default, Deserialize, Serialize, Hash)]
+#[derive(Eq, PartialEq, Clone, Debug, Default, Deserialize, Serialize)]
```

Comment:
Nice. I don't think Trade should ever have been `Hash` - it doesn't really make sense.

Reply:
I recall you were saying this and finally figured out how to get rid of it.